docs/source/QUICKSTART.md

```diff
@@ -0,0 +1,99 @@
+# Quick Start Guide
+
+This guide will help you get started with SymbolicAI, demonstrating basic usage and key features.
+
+To start, import the library by using:
+
+```python
+import symai as ai
+```
+
+## Creating and Manipulating Symbols
+
+Our `Symbolic API` is based on object-oriented and compositional design patterns. The `Symbol` class serves as the base class for all functional operations, and in the context of symbolic programming (fully resolved expressions), we refer to it as a terminal symbol. The Symbol class contains helpful operations that can be interpreted as expressions to manipulate its content and evaluate new Symbols `<class 'symai.expressions.Symbol'>`.
+
+```python
+# Create a Symbol
+sym = ai.Symbol("Welcome to our tutorial.")
+
+# Translate the Symbol
+translated = sym.translate('German')
+print(translated)  # Output: Willkommen zu unserem Tutorial.
+```
+
+### Ranking Objects
+
+Our API can also execute basic data-agnostic operations like `filter`, `rank`, or `extract` patterns. For instance, we can rank a list of numbers: #TODO linking of functions
+
+```python
+# Ranking objects
+import numpy as np
+sym = ai.Symbol(np.array([1, 2, 3, 4, 5, 6, 7]))
+ranked = sym.rank(measure='numerical', order='descending')
+print(ranked)  # Output: ['7', '6', '5', '4', '3', '2', '1']
+```
+
+### Evaluating Expressions
+
+Evaluations are resolved in the language domain and by best effort. We showcase this on the example of [word2vec](https://arxiv.org/abs/1301.3781).
+
+**Word2Vec** generates dense vector representations of words by training a shallow neural network to predict a word based on its neighbors in a text corpus. These resulting vectors are then employed in numerous natural language processing applications, such as sentiment analysis, text classification, and clustering.
+
+In the example below, we can observe how operations on word embeddings (colored boxes) are performed. Words are tokenized and mapped to a vector space where semantic operations can be executed using vector arithmetic.
+
+<img src="https://raw.githubusercontent.com/ExtensityAI/symbolicai/main/assets/images/img3.png" width="450px">
+
+Similar to word2vec, we aim to perform contextualized operations on different symbols. However, as opposed to operating in vector space, we work in the natural language domain. This provides us the ability to perform arithmetic on words, sentences, paragraphs, etc., and verify the results in a human-readable format.
+
+The following examples display how to evaluate such an expression using a string representation:
+
+```python
+# Word analogy
+result = ai.Symbol('King - Man + Women').expression()
```

Comment:
This is `.interpret()`. `.expression` became a decorator and it's used in `wolframalpha` interface.